python/Makefile.mk

```diff
@@ -3,11 +3,11 @@ include python/ola/Makefile.mk
 
 python/PyCompileTest.sh: python/Makefile.mk
 	mkdir -p $(top_builddir)/python
-# 	restore this line when py3 compat is done for whole tree
-#	echo "$(PYTHON) -m compileall -f tools scripts python include data; exit \$$?" > $(top_builddir)/python/PyCompileTest.sh
-	echo "$(PYTHON) -m compileall python data; exit \$$?" > $(top_builddir)/python/PyCompileTest.sh
+	echo "$(PYTHON) -m compileall -f $(PYTHON_BUILD_DIRS); exit \$$?" > $(top_builddir)/python/PyCompileTest.sh
 	chmod +x $(top_builddir)/python/PyCompileTest.sh
```

Comment:
I still wonder if this should be in Makefile.am, but don't feel too strongly either way, the variable is certainly a great improvement.